tools/issue_to_scripts.py

```
@@ -0,0 +1,279 @@
+    if not _is_empty(raw_description):
+        entry["description"] = raw_description
+
+    raw_infourl = sections.get("Info URL", "").strip()
+    if not _is_empty(raw_infourl):
+        if not raw_infourl.startswith(("http://", "https://")):
+            raise ValueError(f"Info URL must start with http:// or https://: {raw_infourl!r}")
+        entry["infourl"] = raw_infourl
+
+    raw_images = sections.get("Image URLs", "")
+    if not _is_empty(raw_images):
+        lines = [l.strip() for l in raw_images.splitlines() if l.strip()]
```

Ruff (0.15.12)

[error] Ambiguous variable name: `l`

(E741)